install-gui.sh

```diff
@@ -40,17 +40,17 @@ do_install_npm_locally() {
   NODEJS_VERSION="$(node -v | cut -d'.' -f 1 | sed -e 's/^v//')"
   NPM_VERSION="$(npm -v | cut -d'.' -f 1)"
 
-  if [ "$NODEJS_VERSION" -lt "18" ] || [ "$NPM_VERSION" -lt "9" ]; then
-    if [ "$NODEJS_VERSION" -lt "18" ]; then
-      echo "Current NodeJS version($(node -v)) is less than 18. GUI app requires NodeJS>=18."
+  if [ "$NODEJS_VERSION" -lt "20" ] || [ "$NPM_VERSION" -lt "10" ]; then
+    if [ "$NODEJS_VERSION" -lt "20" ]; then
+      echo "Current NodeJS version($(node -v)) is less than 20. GUI app requires NodeJS>=20."
     fi
     if [ "$NPM_VERSION" -lt "9" ]; then
```

Comment:
This should be `if [ "$NPM_VERSION" -lt "10" ]; then`

Reply:
updated